chapters/ch40.md

```diff
@@ -1,104 +1,102 @@
 
---> _replace THIS LINE by your translation for the above line_
+--> Bạn giờ đây có bốn tập con dữ liệu:
 
 
 > * Training set. This is the data that the algorithm will learn from (e.g., Internet images + Mobile images). This does not have to be drawn from the same distribution as what we really care about (the dev/test set distribution).
 
---> _replace THIS LINE by your translation for the above line_
+--> * Tập huấn luyện: Đây là dữ liệu mà thuật toán sẽ học từ nó (ví dụ: ảnh Internet + ảnh Điện Thoại). Tập dữ liệu này không nhất thiết phải được trích xuất từ cùng phân phối như là đối với tập phát triển/kiểm tra.
 
 
 > * Training dev set: This data is drawn from the same distribution as the training set (e.g., Internet images + Mobile images). This is usually smaller than the training set; it only needs to be large enough to evaluate and track the progress of our learning algorithm.
 
---> _replace THIS LINE by your translation for the above line_
+--> * Tập huấn luyện phát triển: Đây là dữ liệu trích xuất từ cùng phân phối với tập huấn luyện (ví dụ: ảnh Internet + ảnh Điện Thoại). Nó thông thường nhỏ hơn tập huấn luyện và chỉ cần đủ lớn để có thể đánh giá và theo dõi quá trình học của thuật toán.
 
 
 > * Dev set: This is drawn from the same distribution as the test set, and it reflects the distribution of data that we ultimately care about doing well on. (E.g., mobile images.)
 
---> _replace THIS LINE by your translation for the above line_
+--> * Tâp phát triển: Đây là dữ liệu trích xuất từ cùng phân phối với tập kiểm tra, nó phản ánh phân phối dữ liệu mà chúng ta mong muốn thuật toán thực hiện tốt nhất. (Ví dụ: ảnh điện thoại)
 
 
 > * Test set: This is drawn from the same distribution as the dev set. (E.g., mobile images.)
 
---> _replace THIS LINE by your translation for the above line_
+--> * Tập kiểm tra: Đây là dữ liệu trích xuất từ cùng phân phối với tập phát triển. (Ví dụ: ảnh điện thoại)
 
 
 > Armed with these four separate datasets, you can now evaluate:
 
---> _replace THIS LINE by your translation for the above line_
+--> Được trang bị với bốn tập dữ liệu riêng biệt, bạn giờ đây có  thể đánh giá:
 
 
 > * Training error, by evaluating on the training set.
 
---> _replace THIS LINE by your translation for the above line_
+--> * Lỗi huấn luyện, bằng cách đánh giá tập huấn luyện.
 
 
 > * The algorithm’s ability to generalize to new data drawn from the training set distribution, by evaluating on the training dev set.
 
---> _replace THIS LINE by your translation for the above line_
+--> * Khả năng tổng quát hóa của thuật toán đối với dữ liệu mới trích xuất từ cùng phân phối với tập huấn luyện, bằng cách đánh giá tập huấn luyện phát triển.
 
 
 > * The algorithm’s performance on the task you care about, by evaluating on the dev and/or test sets.
 
---> _replace THIS LINE by your translation for the above line_
+--> * Chất lượng của thuật toán trên nhiệm vụ mà bạn quan tâm, bằng cách đánh giá tập phát triển và/hoặc tập kiểm tra.
```

Comment:
~~Chất lượng~~Hiệu quả của thuật toán
> Như thông nhất cách dịch với các chương khác.

trên ~~nhiệm vụ~~tác vụ mà bạn quan tâm

Reply:
Từ "tác vụ" này  đọc xuôi tai hơn hẳn. Cảm ơn bạn.